### PR TITLE
fix(TBD-15282): Correct label for HDI

### PR DESCRIPTION
**Pull request by sponomarova**
**Please check if the PR fulfills these requirements**

- [ ] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format) ?
- [ ] Unit tests for the Java changes have been added (for bug fixes / features) ?
- [ ] TUJ for the JavaJet changes have been added (for bug fixes / features) ?
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The new code does not introduce new technical issues

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**

**BREAKING CHANGE**

If this PR contains a breaking change, please describe the impact and migration
path for existing applications.
If not please remove this section.

**Other information**:

### CODE REVIEW

**Review by sanulich — DISMISSED** [review]
LGFM

File: main/plugins/org.talend.hadoop.distribution/src/main/java/messages.properties [hunk 1]
```diff
@@ -42,7 +42,7 @@ ESparkMode.SPARK_LOCAL=Local
 ESparkMode.CDE=Cloudera Data Engineering
 ESparkMode.DATAPROC=Dataproc
 ESparkMode.SYNAPSE=Synapse
-ESparkMode.HDI=HD Insight
+ESparkMode.HDInsight=HDInsight
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/messages.properties:45]:
```suggestion
ESparkMode.HDInsight=HDInsight
ESparkMode.HDINSIGHT=HDInsight
```
If you really want to change from HDI on key part we should stay with capital letters to be consistent

File: main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/apache/ESparkMode.java [hunk 2]
```diff
@@ -47,7 +47,7 @@ public enum ESparkMode {
     STANDALONE(Messages.getString("ESparkMode.STANDALONE"),
             "STANDALONE",
             "(DISTRIB[DISTRIBUTION, SPARK_VERSION].doSupportUniversalStandaloneMode[])"),
-    HDI(Messages.getString("ESparkMode.HDI"),
+    HDI(Messages.getString("ESparkMode.HDInsight"),
```
Comment (lbourgeois) [inline main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/apache/ESparkMode.java:50]:
```suggestion
    HDI(Messages.getString("ESparkMode.HDInsight"),
    HDI(Messages.getString("ESparkMode.HDINSIGHT"),
```

Comment (sponomarova) [reply main/plugins/org.talend.hadoop.distribution/src/main/java/org/talend/hadoop/distribution/constants/apache/ESparkMode.java]: Agree. Changed.

**Review by sanulich — APPROVED** [review]

**Pull request merged**